plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/parameters/GenerateClientParameters.kt

```diff
@@ -37,7 +37,7 @@ interface GenerateClientParameters : WorkParameters {
     /** Type of JSON serializer that will be used to generate the data classes. */
     val serializer: Property<GraphQLSerializer>
     /** GraphQL schema file that will be used to generate client code. */
-    val schemaFile: Property<File>
+    val schemaFile: Property<String>
```

Comment:
lets also update javadoc to reflect this is a path to a schema file

```suggestion
    val schemaPath: Property<String>
```